### DISCUSSION TITLE
Binary compatibility break ?

### DISCUSSION THREAD

**Discussion by nbauma109** [eclipse-jdt/eclipse.jdt.core#2337] [Q&A] [answered]
Hi,

I'm trying to port a plugin to the newest version of Eclipse 2024-03.

I encountered the following issue :

`java.lang.NoSuchMethodError: 'java.lang.Object org.eclipse.jdt.internal.core.BinaryType.getElementInfo()'`

Please help to find the proper replacement for this one.

Thanks.

**Comment by iloveeclipse** [comment, marked as answer]
See https://github.com/eclipse-jdt/eclipse.jdt.core/blob/874f9332ef1175ac5570159d41a6b39cb9d77152/org.eclipse.jdt.core/model/org/eclipse/jdt/internal/core/BinaryType.java#L1161

It is **not** API.
